.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "opentelemetry-otlp/src/proto/opentelemetry-proto"]
```

Comment:
Even when it does stabilize, it might be cleaner to just keep it as a submodule and update the commit it points to, that way it's clear what version of the proto this points to. I tried to follow a similar pattern to the [go sdk](https://github.com/open-telemetry/opentelemetry-go/tree/master/exporters/otlp/internal), which adds `opentelemetry-proto` as a submodule. Though that just references the generated go code in the module.